packages/suite-native-settings/src/navigation/routes.ts

```diff
@@ -0,0 +1,9 @@
+export enum SettingsRouteStack {
+    Settings = 'Settings',
+    SettingsDetail = 'SettingsDetail',
+}
+
+export type SettingsStackParamList = {
+    [SettingsRouteStack.Settings]: undefined;
+    [SettingsRouteStack.SettingsDetail]: { msg: string };
```

Comment:
Please don't use shortcuts at all msg => message, I know this is just example but then someone will copy that.